clients/src/main/java/org/apache/kafka/clients/consumer/internals/MembershipManagerImpl.java

```diff
@@ -1160,7 +1160,7 @@ private CompletableFuture<Void> assignPartitions(
 
         // Invoke user call back.
         CompletableFuture<Void> result = invokeOnPartitionsAssignedCallback(addedPartitions);
-        result.whenComplete((error, callbackResult) -> {
+        result.whenComplete((callbackResult, error) -> {
```

Comment:
nit: Should we rename `error` to `exception` or should we replace `callbackResult` by `__` as we don't use it?

Reply:
Sure, done